variables.tf

```diff
@@ -4,12 +4,30 @@ variable "create_iam_account" {
   default     = true
 }
 
+variable "create_accessanalyzer" {
+  description = "Controls whether to configure the IAM Access Analyzer"
+  type        = bool
+  default     = false
+}
+
 variable "account_alias" {
   description = "Name of the IAM account alias"
   type        = string
   default     = ""
 }
 
+variable "analyzer_name" {
+  description = "Name of the Analyzer."
+  type        = string
+  default     = "AccountAnalyzer"
```

Comment:
Is this the default that AWS fills in if you go through the console or something? Just curious as to whether there is more rationale behind the default value...

Reply:
When created in the console, you can override the name or accept the default of "ConsoleAnalyzer+[dynamic resource id]". I figured it makes sense to drop the "Console" part....since this resource has nothing to do with the Console, and the analyzer "type" is "ACCOUNT". The unique id doesn't provide value that I can tell, and we'd have to parse the id out of self.arn (I'm allergic to regex so I didn't do this). Since the name attribute is required in Terraform/API, we can't blindly accept the default provided by the Web Console experience.